sigh/lib/sigh/runner.rb

```diff
@@ -29,6 +29,8 @@ def run
           profile.delete!
           profile = create_profile!
         end
+      elsif Sigh.config[:skip_profile_creation]
+        UI.user_error!("No existing profiles found, that match the certificates you have installed locally!")
```

Comment:
Maybe add an indication in the error message that this is ok as indicated with a param/option?